hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/TestAbfsConfigurationFieldsValidation.java

```diff
@@ -182,4 +182,11 @@ public void testSSLSocketFactoryConfiguration()
     assertEquals(DelegatingSSLSocketFactory.SSLChannelMode.OpenSSL, localAbfsConfiguration.getPreferredSSLFactoryOption());
   }
 
+  public static AbfsConfiguration updateRetryConfigs(AbfsConfiguration abfsConfig,
```

Comment:
Not sure if this class is the right place for this method.

Reply:
This is the unit test class for AbfsConfiguration. I do not want to create separate utility class just to return a test instance of main class.